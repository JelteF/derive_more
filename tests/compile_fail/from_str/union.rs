#[derive(derive_more::FromStr)]
union IntOrFloat {
    i: u32,
}

fn main() {}
