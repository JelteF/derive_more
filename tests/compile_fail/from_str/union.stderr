error: `FromStr` cannot be derived for unions
 --> tests/compile_fail/from_str/union.rs:2:1
  |
2 | union IntOrFloat {
  | ^^^^^
